tests/test.py

```diff
@@ -1,30 +1,72 @@
-from asaps.models import Client
+import requests_mock
+from asaps import models
+from asnake.client import ASnakeClient
+import pytest
 
 
-def test_createclient():
-    """Test createclient function."""
-    secfilelist = ['secretsProd', 'secretsDev']
-    for secfile in secfilelist:
-        print(secfile)
-        Client.createclient(secfile)
-        assert Client.authclient.get('repositories').status_code == 200
+@pytest.fixture
+def as_ops():
+    client = ASnakeClient(baseurl='https://example.com', username='test',
```

Comment:
When I'm using requests_mock I usually use a completely fake URL like `mock://example.com`. This way, if you happen mess up your request matcher you don't accidentally make a real HTTP request.